tasks/gene_typing/task_amrfinderplus.wdl

```diff
@@ -0,0 +1,137 @@
+version 1.0
+
+task amrfinderplus_nuc {
+  input {
+    File assembly
+    String samplename
+    # Parameters 
+    # --indent_min Minimum DNA %identity [0-1]; default is 0.9 (90%) or curated threshold if it exists
+    # --mincov Minimum DNA %coverage [0-1]; default is 0.5 (50%)
+    String? organism # make optional?
+    Int? minid
+    Int? mincov
+    Int cpu = 4
+    String docker = "quay.io/staphb/ncbi-amrfinderplus:3.10.24"
+  }
+  command <<<
+    # logging info
+    date | tee DATE
+    amrfinder --version | tee AMRFINDER_VERSION
+    
+    ### set $amrfinder_organism BASH variable based on gambit_predicted_taxon or user-defined input string
+    ### final variable has strict syntax/spelling based on list from amrfinder --list_organisms
+    # there may be other Acinetobacter species to add later, like those in the A. baumannii-calcoaceticus species complex
+    if [[ "~{organism}" == *"Acinetobacter_baumannii"* ]] || [[ "~{organism}" == *"Acinetobacter baumannii"* ]]; then
```

Comment:
adjust these string matches to use `Acinetobacter*baumannii` to catch both spaces, underscores, dashes or whatever.

Reply:
for all organisms

Reply:
fixed